torch/_inductor/codegen/cpp.py

```diff
@@ -3713,8 +3698,30 @@ def is_lowp_fp_scheduler(self, scheduler_node: SchedulerNode):
 
     def legalize_lowp_fp_dtype_loopbody(self, loop_body: LoopBody):
         def add_to_dtype(sub_graph: torch.fx.Graph):
+            def is_dtype_source(node: torch.fx.Node):
+                dtype_sources = [
+                    "load",
+                    "constant",
+                    "to_dtype",
+                    "to_dtype_bitcast",
+                ]
+                return node.target in dtype_sources
+
+            def is_dtype_sink(node: torch.fx.Node):
```

Comment:
Ditto

Reply:
Renamed as `is_lowp_fp_sink` and added some comments